Lesson_11_module/user_input.py

```diff
@@ -0,0 +1,12 @@
+def temperature_get() ->float:
+    temperature_str = input("Please enter degree of temperature to convert: ")
+    if not temperature_str.isdigit():
+        raise ValueError('the input has to be only numbers')
+    return float(temperature_str)
+
+def conversion_type_get():
+    conversion_type = input("Please select conversion type:"
+                            " \n('1' for Celsius to Fahrenheit, '2' for Fahrenheit to Celsius): ")
```

Comment:
The function shouldn't have input() inside. The function should have arguments that you should send when calling the function